app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
Inherit the `Elf` class from `ABC` since it does not implement all abstract methods of the `Player` thus being an abstract class as well.